test/e2e/storage/testsuites/provisioning.go

```diff
@@ -335,16 +341,9 @@ func (t StorageClassTest) TestDynamicProvisioning() *v1.PersistentVolume {
 // persistent across pods.
 //
 // This is a common test that can be called from a StorageClassTest.PvCheck.
-func PVWriteReadSingleNodeCheck(client clientset.Interface, claim *v1.PersistentVolumeClaim, volume *v1.PersistentVolume, node framework.NodeSelection) {
-	By(fmt.Sprintf("checking the created volume is writable and has the PV's mount options on node %+v", node))
```

Comment:
Why was the mount check moved to the reader pod?

Reply:
The mount check required validating the mountOptions from the PV Object.  However on first pod creation with delayed binding, we don't have the PV yet.